openshift/openshift.go

```diff
@@ -226,7 +227,7 @@ func (s *openshiftImageSource) GetBlob(info types.BlobInfo) (io.ReadCloser, int6
 	return s.docker.GetBlob(info)
 }
 
-func (s *openshiftImageSource) GetSignatures() ([][]byte, error) {
+func (s *openshiftImageSource) GetSignatures(ctx context.Context) ([][]byte, error) {
```

Comment:
Could this one be implemented now?

I’m perfectly fine with having `context` support missing in most of the API, but when it exists, it would be better for users if it reliably worked.